src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java

```diff
@@ -136,7 +136,7 @@ public String run(@Nonnull EnvVars launchEnv, @Nonnull String image, @CheckForNu
      * @param containerId The container ID.
      */
     public void stop(@Nonnull EnvVars launchEnv, @Nonnull String containerId) throws IOException, InterruptedException {
-        LaunchResult result = launch(launchEnv, false, "stop", containerId);
+        LaunchResult result = launch(launchEnv, false, "stop", "--time=1", containerId);
```

Comment:
I would recommend using `kill` this will stop the container immediately instead of waiting a second and killing it then.